src/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -332,23 +340,61 @@ void LayerTestsCommon::Compare(const InferenceEngine::TensorDesc &actualDesc, co
     ASSERT_EQ(actualDesc.getPrecision(), expectedDesc.getPrecision());
 }
 
+void LayerTestsCommon::AlignParameters() {
+    int gapInLayout = cnnNetwork.getInputsInfo().size() - inLayout.size();
```

Comment:
```
if (inLayout.size() == 1) {
    for (size_t i = 1; i < cnnNetwork.getInputsInfo().size(); ++i) {
        inLayout.push_back(inLayout.front());
}
```
And the same for other

Reply:
Fixed in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/e0e8cd6fa823f4e857324a52d226c384a26f7547 and https://github.com/openvinotoolkit/openvino/pull/9975/commits/8687c953072c9d814814a8254251814effb351fa.